chains/ethereum/listener_test.go

```diff
@@ -99,21 +99,17 @@ func TestListener_Erc20DepositedEvent(t *testing.T) {
 	// For debugging
 	go watchEvent(l.conn, utils.Deposit)
 
-	// Get transaction ready
-	opts, _, err := l.conn.newTransactOpts(big.NewInt(0), big.NewInt(DefaultGasLimit), big.NewInt(DefaultGasPrice))
-	if err != nil {
-		t.Fatal(err)
-	}
+	client := ethtest.NewClient(t, TestEndpoint, AliceKp)
 
-	erc20Contract := ethtest.DeployMintApproveErc20(t, l.conn.conn, opts, contracts.ERC20HandlerAddress, big.NewInt(100))
+	erc20Contract := ethtest.DeployMintApproveErc20(t, client, contracts.ERC20HandlerAddress, big.NewInt(100))
```

Comment:
@ansermino in `DeployMintApproveErc20`, should I be passing in `l.conn.conn` or `client`?

Reply:
`client` is fine. 

The `ethclient.Client` for these are definitely identical. The `opts` should be the same, it's possible they differ but I think these are both use the `Alice` key.